tests/unit/dataactcore/factories/job.py

```diff
@@ -110,6 +110,14 @@ class Meta:
     narrative = fuzzy.FuzzyText()
 
 
+class ApplicationTypeFactory(factory.Factory):
+    class Meta:
+        model = jobModels.ApplicationType
+
+    application_type_id = fuzzy.FuzzyInteger(9999)
+    application_name = fuzzy.FuzzyText()
+
+
 class WindowFactory(factory.Factory):
```

Comment:
Why is this called `WindowFactory` when the model is `SubmissionWindow`? It should be `SubmissionWindowFactory`